test/e2e/fixture/fixture.go

```diff
@@ -136,22 +133,15 @@ func EnsureCleanState() {
 
 	start := time.Now()
 
+	policy := v1.DeletePropagationBackground
 	// delete resources
-	text, err := Run("", "kubectl", "get", "app", "-o", "name")
-	CheckError(err)
-	for _, name := range strings.Split(text, "\n") {
-		if name != "" {
-			appName := strings.TrimPrefix(name, "application.argoproj.io/")
-			// we cannot delete any app, if an op is in progress
-			_, _ = RunCli("app", "terminate-op", appName)
-			// is it much more reliable to get argocd to delete an app than kubectl, deleting directly can make ArgoCD stuck
-			_, _ = RunCli("app", "delete", appName)
-		}
-	}
-	FailOnErr(Run("", "kubectl", "delete", "appprojects", "--field-selector", "metadata.name!=default"))
-	// takes around 5s, so we don't wait
+	CheckError(AppClientset.ArgoprojV1alpha1().Applications(ArgoCDNamespace).DeleteCollection(&v1.DeleteOptions{PropagationPolicy: &policy}, v1.ListOptions{}))
```

Comment:
Maybe comment with the kubectl equivilant?

Reply:
sure. added